features/support/run_helpers.rb

```diff
@@ -30,8 +30,8 @@ def git_town_command? command
 end
 
 
-def output_of command
-  run(command).out.strip
+def output_of command, allow_failures: false
+  run(command, allow_failures: allow_failures).out.strip
```

Comment:
All these changes are better reverted before you merge Charlies commits, otherwise you'll run into pretty substantial merge conflicts. Once his changes are in, I recommend you do a `git reset master`, then a re-commit of everything you want to keep, and reverting the rest.